ros-one.repos

```diff
@@ -341,28 +349,32 @@ repositories:
     version: master
   jsk_common:
     type: git
-    url: https://github.com/ros-o/jsk_common.git
-    version: obese-devel
+    url: https://github.com/mqcmd196/jsk_common.git
+    version: techfak-ros-o
```

Comment:
Your private fork strongly deviates from the official ones. Please, merge your changes into `jsk-ros-pkg/jsk_common` or `ros-o/jsk_common`.

Reply:
https://github.com/ubi-agni/ros-builder-action/pull/36/commits/bef463103dcde5d05a91b717cf996a6e2a3a062c